### PR TITLE
Add proxy autostart/stop to full-reference test fly.toml

### PR DESCRIPTION
**Pull request by catflydotio**
Add `auto_stop_machines`, `auto_start_machines`, and `min_machines_running` sample values to `internal/appconfig/testdata/full-reference.toml`

### CODE REVIEW

**Comment by dangra** [comment]
Oh, you jumped into a rabbithole but hopefully it is full of examples to follow.

Run `go test ./internal/appconfig/` until it passes.

File: internal/appconfig/testdata/full-reference.toml [hunk 1]
```diff
@@ -128,7 +131,10 @@ host_dedication_id = "06031957"
   internal_port = 8081
   protocol = "tcp"
   processes = ["app"]
-
+  auto_start_machines = false
+  auto_stop_machines = false
+  min_machines_running = 1
+
```
Comment (catflydotio) [inline internal/appconfig/testdata/full-reference.toml:137]:
```suggestion

```

**Comment by catflydotio** [comment]
I marked this ready for review because now I think the only tests failing are from transient errors creating volumes.

**Review by dangra — APPROVED** [review]
💯

**Pull request merged**